scripts/plopfile.js

```diff
@@ -66,9 +67,9 @@ module.exports = function (plop) {
         if (scopes.includes('@react-aria')) {
           actions.push({
             type: 'addMany',
-            templateFiles: 'plop-templates/@react-aria/**',
-            base: 'plop-templates/@react-aria/',
-            destination: `packages/@react-aria/${packageName}`,
+            templateFiles: '../plop-templates/@react-aria/**',
```

Comment:
huh... interesting, why'd this need to change?

Reply:
Without these changes plop wasn't generating files cuz it couldn't find the template files. I did some googling and found that plop uses paths relative to the plopfile location to find stuff. This used to work without the `../` because the plopfile was at the root of the repo but is now in the scripts folder